be/src/olap/storage_engine.cpp

```diff
@@ -428,6 +428,18 @@ std::vector<DataDir*> StorageEngine::get_stores_for_create_tablet(
     std::random_device rd;
     srand(rd());
     std::random_shuffle(stores.begin(), stores.end());
+    // Two random choices
+    for (int i = 0; i < stores.size(); i++) {
+        int j = i + 1;
+        if (j < stores.size()) {
+            if (stores[i]->tablet_set().size() > stores[j]->tablet_set().size()) {
+                std::swap(stores[i], stores[j]);
+            }
+            std::random_shuffle(stores.begin() + j, stores.end());
```

Comment:
What's the purpose of the this random_shuffle?

Reply:
@chaoyli 
The purpose of the this random_shuffle is that:
       If it failed to create tablet on the first disk selected using "two random choices", it is necessary to select another disk from the remaining disks to create the tablet by using "two random choices" again. This random_shuffle is the beginning of the next "two random choices".